SPECS/gcc/gcc.spec

```diff
@@ -182,7 +182,7 @@ make %{?_smp_mflags} check-gcc
 %files -n libgcc-devel
 %defattr(-,root,root)
 %{_lib64dir}/libgcc_s.so
-%{_lib}/libcc1.*
+%{_lib64dir}/libcc1.*
```

Comment:
Thanks for looking into this @aostapenko! The reason this is the way you see it, is due to a slight build environment difference between the toolchain builds (they build packages like `gcc`, `python3`, `binutils`, etc.) and "regular" package builds. In case of our pipeline builds, `gcc` only gets built as a toolchain package, where the `%{_lib}/libcc1.*` entry is actually correct and the package published to our RPM repos looks like this:

```
root [ / ]# dnf -q repoquery -l libgcc-devel
/usr/lib/libcc1.la
/usr/lib/libcc1.so
/usr/lib/libcc1.so.0
/usr/lib/libcc1.so.0.0.0
/usr/lib64/libgcc_s.so
```

We do want to get these two builds aligned but this will require a lot more work than just this one change, which would currently break the toolchain builds.

@anphel31: please correct me, if I got something wrong here.

Reply:
Could you please describe your build environment. Given all the guides are written for Ubuntu 18.04 I would expect this to work there. I tried both 18.04 and 20.04.
I also have couple of fixes not pushed that I required for 18.04